iterator/snapshot_iterator.go

```diff
@@ -3,26 +3,33 @@ package iterator
 import (
 	"context"
 	"fmt"
-	"github.com/AdamHaffar/conduit-connector-airtable/config"
-	"github.com/AdamHaffar/conduit-connector-airtable/position"
+	"github.com/conduitio-labs/conduit-connector-airtable/config"
+	"github.com/conduitio-labs/conduit-connector-airtable/position"
 	sdk "github.com/conduitio/conduit-connector-sdk"
 	airtableclient "github.com/mehanizm/airtable"
+	"time"
 )
 
 type SnapshotIterator struct {
-	client      *airtableclient.Client
-	data        *airtableclient.Records
-	internalPos position.Position
+	client   *airtableclient.Client
+	data     *airtableclient.Records
```

Comment:
`data` might be a bit generic in this context... What do you think about `records`?